packages/xo-server-sdn-controller/src/index.js

```diff
@@ -1070,9 +1074,8 @@ class SDNController extends EventEmitter {
   // ---------------------------------------------------------------------------
 
   _createOvsdbClient(host) {
-    const foundClient = this.ovsdbClients[host.$ref]
-    if (foundClient !== undefined) {
-      return foundClient
+    if (this.ovsdbClients[host.$ref] !== undefined) {
+      return
```

Comment:
This is not chore, this is a behavior change, why?

Reply:
Not really because the return value has never been used.

Reply:
Ok 👍